library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -1169,22 +1142,13 @@ def GetDevTypeDescription(type, longdescr)
           _("Serial Line Connection")
         ],
         # Device type label
-        "tr"    => [
-          _("Token Ring"),
-          _("Token Ring Network Card")
-        ],
-        # Device type label
-        "usb"   => [_("USB"), _("USB Network Device")],
-        # Device type label
         "vmnet" => [_("VMWare"), _("VMWare Network Device")],
         # Device type label
         "wlan"  => [
           _("Wireless"),
           _("Wireless Network Card")
         ],
         # Device type label
-        "xp"    => [_("XPNET"), _("XP Network")],
-        # Device type label
         "vlan"  => [_("VLAN"), _("Virtual LAN")],
         # Device type label
         "br"    => [_("Bridge"), _("Network Bridge")],
```

Comment:
I see in this list more opurtinities for cleaning :)

Reply:
I'm not sure about getting rid of the descriptions at all. The important part is to determining which are not supported for skipping them from the interfaces list as well for preventing them to be chosen when configuring the network.

Reply:
In my POV it should, or already is, be replaced by InterfaceType in the yast2-network. I really don't see (in fact have never seen) a reason to have the description in NetworkInterfaces.

Reply:
See https://github.com/yast/yast-network/pull/1009